### ISSUE TITLE
IndexError exception on running cli

### ISSUE THREAD

**Issue by dennypenta** [telepresenceio/telepresence#1062]
### What were you trying to do?

shell telepresence

### What did you expect to happen?

absolutely anything else

### What happened instead?

Traceback (most recent call last):
  File "/usr/local/bin/telepresence/telepresence/cli.py", line 136, in crash_reporting
    yield
  File "/usr/local/bin/telepresence/telepresence/main.py", line 60, in main
    remote_info = start_proxy(runner)
  File "/usr/local/bin/telepresence/telepresence/proxy/__init__.py", line 102, in start_proxy
    run_id=run_id,
  File "/usr/local/bin/telepresence/telepresence/proxy/remote.py", line 153, in get_remote_info
    runner, deployment_name, deployment_type, run_id=run_id
  File "/usr/local/bin/telepresence/telepresence/proxy/remote.py", line 100, in get_deployment_json
    )["items"][0]
IndexError: list index out of range

Here are the last few lines of the logfile (see /Users/dvornikov/gomods/gateway/telepresence.log for the complete logs):

  22.7  12 | No resources found
  22.7 TEL | [12] ran in 1.61 secs.
  22.7 TEL | [13] Running: oc --context mtp-testing/platform-preprod-tutu-ru:443/dvornikov --namespace mtp-testing run --restart=Always --limits=cpu=100m,memory=256Mi --requests=cpu=25m,memory=64Mi telepresence-1561389929-964444-49693 --image=datawire/telepresence-k8s:0.101 --labels=telepresence=1090019c9c9e4a3caf154707472fded5
  22.9  13 | kubectl run --generator=deploymentconfig/v1 is DEPRECATED and will be removed in a future version. Use kubectl run --generator=run-pod/v1 or kubectl create instead.
  24.8  13 | deploymentconfig.apps.openshift.io/telepresence-1561389929-964444-49693 created
  24.8 TEL | [13] ran in 2.14 secs.
  24.8 TEL | END SPAN deployment.py:97(create_new_deployment)    3.8s
  24.9 TEL | BEGIN SPAN remote.py:151(get_remote_info)
  24.9 TEL | BEGIN SPAN remote.py:78(get_deployment_json)
  24.9 TEL | [14] Capturing: oc --context mtp-testing/platform-preprod-tutu-ru:443/dvornikov --namespace mtp-testing get deployment -o json --selector=telepresence=1090019c9c9e4a3caf154707472fded5
  26.1 TEL | [14] captured in 1.24 secs.
  26.1 TEL | END SPAN remote.py:78(get_deployment_json)    1.2s

### Automatically included information

Command line: `['/usr/local/bin/telepresence']`
Version: `0.101`
Python version: `3.7.3 (default, Mar 27 2019, 09:23:15) 
[Clang 10.0.1 (clang-1001.0.46.3)]`
kubectl version: `Client Version: v1.14.0 // Server Version: v1.11.0+d4cacc0`
oc version: `Client Version: version.Info{Major:"4", Minor:"1+", GitVersion:"v4.1.0+b4261e0", GitCommit:"b4261e07ed", GitTreeState:"clean", BuildDate:"2019-05-18T05:40:34Z", GoVersion:"go1.12.5", Compiler:"gc", Platform:"darwin/amd64"} // Server Version: version.Info{Major:"1", Minor:"11+", GitVersion:"v1.11.0+d4cacc0", GitCommit:"d4cacc0", GitTreeState:"clean", BuildDate:"2018-10-11T07:30:01Z", GoVersion:"go1.10.3", Compiler:"gc", Platform:"linux/amd64"}`
OS: `Darwin admins-MBP-2.office.tutu.ru 18.6.0 Darwin Kernel Version 18.6.0: Thu Apr 25 23:16:27 PDT 2019; root:xnu-4903.261.4~2/RELEASE_X86_64 x86_64`

```
Traceback (most recent call last):
  File "/usr/local/bin/telepresence/telepresence/cli.py", line 136, in crash_reporting
    yield
  File "/usr/local/bin/telepresence/telepresence/main.py", line 60, in main
    remote_info = start_proxy(runner)
  File "/usr/local/bin/telepresence/telepresence/proxy/__init__.py", line 102, in start_proxy
    run_id=run_id,
  File "/usr/local/bin/telepresence/telepresence/proxy/remote.py", line 153, in get_remote_info
    runner, deployment_name, deployment_type, run_id=run_id
  File "/usr/local/bin/telepresence/telepresence/proxy/remote.py", line 100, in get_deployment_json
    )["items"][0]
IndexError: list index out of range

```

Logs:

```
estart=Always --limits=cpu=100m,memory=256Mi --requests=cpu=25m,memory=64Mi telepresence-1561390022-387241-49790 --image=datawire/telepresence-k8s:0.101 --labels=telepresence=9e22a9ff9a0b4756831d94f76a80e245
   4.2  12 | kubectl run --generator=deploymentconfig/v1 is DEPRECATED and will be removed in a future version. Use kubectl run --generator=run-pod/v1 or kubectl create instead.
   5.8  12 | deploymentconfig.apps.openshift.io/telepresence-1561390022-387241-49790 created
   5.8 TEL | [12] ran in 1.63 secs.
   5.8 TEL | END SPAN deployment.py:97(create_new_deployment)    2.1s
   5.8 TEL | BEGIN SPAN remote.py:151(get_remote_info)
   5.8 TEL | BEGIN SPAN remote.py:78(get_deployment_json)
   5.8 TEL | [13] Capturing: oc --context mtp-testing/platform-preprod-tutu-ru:443/dvornikov --namespace mtp-testing get deployment -o json --selector=telepresence=9e22a9ff9a0b4756831d94f76a80e245
   6.9 TEL | [13] captured in 1.15 secs.
   6.9 TEL | END SPAN remote.py:78(get_deployment_json)    1.1s

```

**Comment by ark3** [comment]
Sorry about the crash! This is issue #1010.

**Comment by ark3** [comment]
I believe this was fixed by #1404.
